index.html

```diff
@@ -0,0 +1,261 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./assets/css/style.css">
+    <script src="https://kit.fontawesome.com/8907a964e1.js" crossorigin="anonymous"></script>
+    <title>Josh Holmes Portfolio</title>
+</head>
+<body>
+    <!-- header that contains nav bar -->
+    <header id="home" class="header">
+        <!-- flex container -->
+        <nav class="nav-links-container">
+            <!-- nav link -->
+            <div class="nav-link">
+                <a href="#projects">My Projects</a>
+            </div>
+            <!-- nav link -->
```

Comment:
Whilst comments can be helpful, overcommenting can be counter productive and clogs up the code. Try to mark out the main sections with a comment, rather than describing the individual lines of code.